websites/B/BotsBase/dist/metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+  "author": {
+    "name": "Roxza",
+    "id": "716706383780904961"
+  },
+  "service": "botsbase.net",
```

Comment:
service name and foldername must be the same, and this is the name displayed on our website.